src/Players.js

```diff
@@ -2,18 +2,36 @@ import React, { useContext, useState } from "react";
 import Board from "./Board";
 import { CardContext } from "./CardContext";
 
+let roundSubmits = 0;
+
 export default function Player({ playerid }) {
   const [state, setState] = useContext(CardContext);
+  let playerSubmit = 0;
 
   // get the players hand
   const getHand = (p) => {
     let hand = [];
     if (p === 0) hand = state.playerOne; // player one
-    if (p === 1) hand = state.playerTwo; // player two 
+    if (p === 1) hand = state.playerTwo; // player two
     if (p === 2) hand = state.playerThree; // player three
     if (p === 3) hand = state.playerFour; // player four
     return hand;
-  }
+  };
+
+  // checks and adds number of submits per expedition card round
+  const handlePlayerSubmits = () => {
+    roundSubmits++;
+    playerSubmit++;
+    state.roundSubmits = roundSubmits;
```

Comment:
We can't run with this unfortunately. This is just waiting to break in our faces

Reply:
I see what you mean about state not updating though